### PR TITLE
Xsup 39874 splunk py drilldown

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-39874)

## Description
Fixed an issue where drilldown  query with double double quotes was splitted.
For example "countryName=""" turned to "countryName=""

### CODE REVIEW

**Comment by eyalpalo** [comment]
image

Failed unit test

File: Packs/SplunkPy/ReleaseNotes/3_1_36.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where drilldown search query was parsed wrong.
```
Comment (ShacharKidor) [inline Packs/SplunkPy/ReleaseNotes/3_1_36.md:6]:
```suggestion
- Fixed an issue where drilldown search queries were incorrectly parsed.
```

Comment (dantavori) [inline Packs/SplunkPy/ReleaseNotes/3_1_36.md:6]:
```suggestion
- Fixed an issue where drilldown search queries were parsed incorrectly.
```

**Review by dantavori — APPROVED** [review]

**Review by ShacharKidor — APPROVED** [review]
@eyalpalo Thank you - looks good!
Added a wordings suggestion for the RN file.

**Pull request merged**